README.md

```diff
@@ -20,31 +20,57 @@ Keep in mind that:
 * There is another repo containing the [deprecated version](https://github.com/microg/NetworkLocation) of NetworkLocation.apk without the plug-in system.
 * [microG GmsCore](https://github.com/microg/android_packages_apps_GmsCore/wiki) project already includes the Unified Network Location Provider.
 
-
-### Android 4.4 - 7.1.1 (KitKat / Lollipop / Marshmallow / Nougat)
-Most modern ROMs come with support for non-Google geolocation providers. On these systems installation is easy:
+Most modern ROMs come with support for non-Google geolocation providers.  
+On these systems installation is easy:
 
 1. Make sure that no Google geolocation tool is installed (it is usually listed as Google Play Services in Apps)
 2. Download and install `NetworkLocation.apk` as a usual app (you may need to enable "Unknown sources" in Settings->Security)
 3. Reboot and continue at [Usage](#usage)
 
-Some ROMs, especially those not based on AOSP might have problems using this method. However, if your system has root, you can try installing the hard way:
+**Important:** Installing as system app will always works, instead installing as user app will only works out of the box on Android versions from 4.4 to 6.x.  
```

Comment:
should be "will only work out of the box" (minus the s)

Reply:
Fixed.